src/test/java/com/cdancy/bitbucket/rest/features/RepositoryApiLiveTest.java

```diff
@@ -105,6 +105,25 @@ public void testListRepositories() {
         assertThat(found).isTrue();
     }
 
+    @Test
+    public void testListAllRepositories() {
+        final RepositoryPage repositoryPage = api().listAll(null, null, null, null, 0, 100);
+
+        assertThat(repositoryPage).isNotNull();
+        assertThat(repositoryPage.errors()).isEmpty();
+        assertThat(repositoryPage.size()).isGreaterThan(0);
+
+        assertThat(repositoryPage.values()).isNotEmpty();
+        boolean found = false;
+        for (final Repository possibleRepo : repositoryPage.values()) {
+            if (possibleRepo.name().equals(repoKey)) {
+                found = true;
+                break;
+            }
+        }
+        assertThat(found).isTrue();
+    }
+
```

Comment:
Lets add a live test to exercise the failure case. I guess you _could_ put in say bogus values for things just to get bitbucket to return an error state. I do this for every endpoint so there should be multiple examples within this file.

Reply:
Done. Please review the last commit when you have time.